examples/csv_mail_merge/index.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+var converter = require('json-2-csv');
+var fs       = require('fs');
+var moment   = require('moment');
+var parse    = require('csv-parse');
+
+var lobFactory = require('../../lib/index.js');
+var Lob        = new lobFactory('test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc');
+
+var inputFile = fs.createReadStream(__dirname + '/input.csv');
+var letterTemplate = fs.readFileSync(__dirname + '/letter_template.html').toString();
+
+var companyInfo = {
+  name: 'Deluxe Virgina Realty',
+  address_line1: '185 Berry St.',
+  address_line2: 'Ste 170',
+  address_city: 'San Francsisco',
+  address_state: 'CA',
+  address_zip: 94107,
+  address_country: 'US'
+}
+
+var parser = parse({columns: true}, function (err, data) {
+  if (err) {
+    return console.log(err);
+  }
+
+  data.forEach(function (client) {
+
+    var name = client.name;
+    var amount = client.amount
+    var address = {
+      address_line1: client.address1,
+      address_line2: client.address2,
+      address_city: client.city,
+      address_state: client.state,
+      address_zip: client.zip,
+      address_country: 'US'
+    }
+
+    Lob.verification.verify(address)
+    .then(function (verifiedAddress) {
+      return Lob.letters.create({
+        description: 'Automated Past Due Bill',
```

Comment:
Lets put the client name in the description. "Automated Past Due Bill for Russell"